btas/generic/cp_als.h

```diff
@@ -700,16 +686,15 @@ namespace btas {
       }
       KRP_dims.push_back(ndim);
 
-      contract(1.0, tensor_ref, tref_indices, KhatriRao, KRP_dims, 0.0, temp, An_indices);
+      contract(this-> one , tensor_ref, tref_indices, KhatriRao, KRP_dims, this-> zero, temp, An_indices);
```

Comment:
spaces between `->` and member name as well as between `one` and the comma look weird.